README.md

```diff
@@ -43,15 +47,15 @@ In particular, BenchExec provides three major features:
 
 - execution of arbitrary commands with precise and reliable measurement
   and limitation of resource usage (e.g., CPU time and memory),
-  and isolation against other running processes  
+  and isolation against other running processes 
```

Comment:
Be careful with your editor config. Trailing whitespace is (unfortunately) significant in Markdown and should not be removed (although we do want this in other file types).

Reply:
but these are two trailing whitespace (reduced to one). this shouldn't make a difference I think.

Reply:
Two trailing whitespaces define a line break in Markdown, one trailing whitespace is ignored. So this makes the difference.

(As a side note, please don't mark discussion as resolved that others created. This makes it hard to follow the discussion and making sure that everything is actually resolved.)